### ISSUE TITLE
Watch folder doesn't initiate

### ISSUE THREAD

**Issue by lediszep** [jlesage/docker-handbrake#62]
Hello,

I have a watch folder setup and it seems to work if a file gets dropped in there the first time, but as soon as Radarr drops an upgrade file with the same title in the watch folder Handbrake doesn't encode the file and it just sits there. Does Handbrake keep track of what it already has encoded an there for doesn't initiate the encode due to it seeing it as a file it has already encoded?

Thanks, Shaun

**Comment by jlesage** [comment]
If you add the exact same file twice, HandBrake won't re-encode it.
You can look at the container's log to see what's happening (i.e. why your encoding is skipped).

**Comment by lediszep** [comment]
Any way of disabling that?

**Comment by jlesage** [comment]
Not really...  but is Radarr really put the same video twice?  Or it put a different one with the same file name?

**Comment by lediszep** [comment]
Normally a different filename one would be ... (720p) and the upgrade ... (1080p) but then handbrake doesn't overwrite the file and you end up with both files.

**Comment by jlesage** [comment]
Correct.

So if I understand correctly, Radarr can put in the watch folder 2 different videos (i.e. with different resolutions) but with the same filename.  In this case, I think the conversion may not happen because the output file already exists.  Did you look at the container's log to see why the conversion is not done?

**Comment by dgroe** [comment]
If this is the wrong place to post, I humbly apologize. I just didn't want to create a new issue for what seems like a stupid question.
I've installed the Handbrake docker on my Synology NAS, and have been TRYING to locate (or create) my watch folder. Right now, I'm using what was the default (/watch), which I assume is in the root directory. (I've tried other locations, including my home dir.) 
What am I missing? I've used Unix/Linux since the System V (and BSD) days, but the docker adds a level of indirectness that's throwing me off.  I've used Handbrake on my PC for MANY years, so I have no issue with that functionality. 
Again, apologies if this is not the correct method to post.

**Comment by jlesage** [comment]
How do you create the container? Via the cli?  Which command did you use?
